compute/resource_cluster.go

```diff
@@ -263,6 +276,24 @@ func resourceClusterUpdate(d *schema.ResourceData, m interface{}) error {
 	return resourceClusterRead(d, m)
 }
 
+// wrapClusterRequest helps remove all requests that should not be submitted when instance pool is selected.
+func wrapClusterRequest(clusterModel *Cluster) {
```

Comment:
in this case it's not `wrap` as well - it's not taking something and returning it, it's `adjust` or `modify`.

Reply:
done